Gems/Atom/Feature/Common/Assets/Shaders/PostProcessing/BloomCompositeCS.azsl

```diff
@@ -16,14 +16,15 @@ ShaderResourceGroup PassSrg : SRG_PerPass
     RWTexture2D<float4>   m_outputTexture;
 
     float  m_intensity;
+    uint m_enableBicubic;
```

Comment:
As an alternative, consider moving `m_tint` up above to `m_intensity` (tint and intensity seem more related to me)

Reply:
I am surprised you even need this change since this data is tracked via Constant data of SRG. What happens if you revert his change?

Reply:
reverted this after verifying on updated android version.